plugins/juju/04known_bugs

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
+import logging
+import subprocess
+
+from common import helpers
+from common.constants import JUJU_LOG_PATH
+
+log = logging.getLogger("juju.04known-bugs")
+
+
+def detect_bug1910958():
+    """Unit fails to start complaining there are members in the relation."""
+    result = {"bug": "LP: #1910958",
+              "description": ("Unit fails to start complaining there are "
+                              "members in the relation."),
+              }
+    pattern = (
+        'manifold worker returned unexpected error: failed to initialize '
+        'uniter for "[A-Za-z0-9\\-]+": cannot create relation state tracker: '
+        'cannot remove persisted state, relation \\d+ has members'
+    )
+
+    cmd = ["grep", "-r", "-P", "-H", "--include=*.log", "--files-with-match",
+           pattern, JUJU_LOG_PATH]
+    try:
```

Comment:
1 cent on this, why not making this a util?

Reply:
made this a function that wraps `grep` calls.